includes/admin/settings/register-settings.php

```diff
@@ -1293,12 +1293,12 @@ function edd_settings_sanitize( $input = array() ) {
 					}
 
 					$setting_details = edd_get_registered_setting_details( $tab, $section, $key );
-					$number_type = false !== strpos( $setting_details['step'], '.' ) ? 'floatval' : 'intval';
-					$minimum   = $number_type( $setting_details['min'] );
-					$maximum   = $number_type( $setting_details['max'] );
-					$new_value = $number_type( $input[ $key ] );
+					$number_type     = ! empty( $setting_details['step'] ) && false !== strpos( $setting_details['step'], '.' ) ? 'floatval' : 'intval';
+					$minimum         = ! empty( $setting_details['min'] ) ? $number_type( $setting_details['min'] ) : false;
```

Comment:
```suggestion
					$minimum         = isset( $setting_details['min'] ) ? $number_type( $setting_details['min'] ) : false;
```

To allow `0` minimums.